easysocial/easysocial/groupinvite.php

```diff
@@ -105,23 +95,39 @@ public function delete()
 			switch ($operation)
 			{
 				case 'leave':
+								// Remove the user from the group/page.
+								$res->result->status = $cluster->leave($user->id);
+
+								if (!$res->result->status)
+								{
+									$res->result->status	=	0;
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_INVALID_USER_MESSAGE');
+								}
+								else
+								{
+									// Notify group/page members
+									$cluster->notifyMembers('leave', array('userId' => $my->id));
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_LEAVE_GROUP_MESSAGE');
+									break;
```

Comment:
@Pratikshakhandagale - Do you really need `break` statement inside `else` block or should it be **outside** `else` block